specs/vep-994-jupyter-notebook-integration/README.md

```diff
@@ -155,31 +156,53 @@ No direct changes to the public API.
 
 ## Detailed design
 ### VDK Notebook plugin
- This VDK plugin will provide the functionality to run Jobs which will retrieve Job steps from notebook files instead of .py and .sql files. This plugin can be used alone without the JupyterLab extension.
+ This VDK plugin will provide the functionality to run Jobs contain notebook files instead of .py and .sql files. This plugin can be used alone without the JupyterLab extension.
```

Comment:
I think the original reads better?

Reply:
yes, but here we have not yet provided a proper definition of what Step means that's why I changed it. bUT NOW I see that I  have omitted putting "which" before contain. I will fix it